ejb3/src/main/java/org/jboss/as/ejb3/deployment/EjbDeploymentInformation.java

```diff
@@ -106,10 +108,7 @@ public EJBComponent getEjbComponent() {
     }
 
     public ComponentView getView(String name) {
-        final InjectedValue<ComponentView> value = componentViews.get(name);
-        if (value == null) {
-            throw new IllegalArgumentException("View " + name + " was not found");
-        }
+        final InjectedValue<ComponentView> value = checkNotNullParam(String.valueOf(name), componentViews.get(name));
```

Comment:
String.valueOf() seems redundant.

Reply:
@chengfang thanks for your review. This is intended because the previous version did not null check 'name'. [org.wildfly.common.Assert.checkNotNullParam](https://github.com/wildfly/wildfly-common/blob/master/src/main/java/org/wildfly/common/Assert.java#L47) does check the not-null for the excetion text. If the String name contained null but null is a valid lookup value (i.E. a default comes back) the previous version would perfectly work. If you require me to change that I would do it with two checks:
`checkNotNullParam("name", name);
final InjectedValue<ComponentView> value = name, componentViews.get(name));`
The two check solution is OK, but it changes the behaviour.
Please have a look at it again and advise what is needed here.

Reply:
@boris-unckel thanks for explaining this. I'm fine with your original fix.